test/include/helpers/sceneHelpers.hpp

```diff
@@ -26,7 +26,7 @@ static rgl_entity_t makeEntity(rgl_mesh_t mesh = nullptr)
 	return entity;
 }
 
-static inline void spawnCubeOnScene(const Mat3x4f& transform, std::optional<int> id = std::nullopt)
+static inline rgl_entity_t spawnCubeOnScene(const Mat3x4f& transform, std::optional<int> id = std::nullopt)
```

Comment:
Any particular reason for that? 
If U return entity with cube, then name `spawnCubeOnScene` is little bit unprecise.

Reply:
now this is spawnAndGet, but maybe I'm to sensitive